js/api.js

```diff
@@ -0,0 +1,15 @@
+var $ = require('jquery');
+
+var ServerActionCreator = require('./actions/RepcoinServerActionCreator.js');
+
+module.exports = {
+  getCategories: function() {
+    $.ajax({
+      url: '/api/categories',
+      success: ServerActionCreator.receiveCategories,
```

Comment:
This is where we make our server requests. Basically, we ask the server for data, and, when we get it back successfully, create an action which is going to propagate to all of the stores dependent on this data, in this case, the CategoriesStore.